### PR TITLE
747 - operation tables fields mismatch

### LINKED ISSUE #747: Operation tables field mismatch
In some cases parameters and micheline parameters content is does not match.

### PR DESCRIPTION
**Pull request by piotrkosecki**
Fixed parameters in transaction not being transformed correctly. 
resolves #747

### CODE REVIEW

**Comment by vishakh** [comment]
Could you describe the precise issue and the fix?

**Comment by piotrkosecki** [comment]
@vishakh The issue was that before my changes lenses were modifying every `parameters_micheline` with the first `parameters` that were found in the block. 
After my change, code will be modifying transaction operation in with `parameters_micheline` from the same transaction.

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala [hunk 1]
```diff
@@ -645,14 +645,21 @@ class TezosNodeOperator(
     val proposalsStateFetch =
       fetchMerge(currentQuorumFetcher, currentProposalFetcher)(CurrentVotes.apply)
 
-    def parseMichelsonScripts: Block => Block = { block: Block =>
+    def parseMichelsonScripts: Block => Block = {
       implicit lazy val _ = logger
+      def rewriteParametersInTransactions(transaction: Transaction): Transaction = {
+        val expr = transaction.parameters.map {
+          case Left(value) => value.value.expression
```
Comment (ivanopagano) [inline src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala:652]: I'd suggest pattern matching on `value` too, to help when reading the code

**Comment by ivanopagano** [comment]
Another thing I plan to do, related to this issue, is to put a wiki entry to show examples of how the most common optics work, which will come handy when working with this parts of the code.

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala (hunk 1 above)
Comment (piotrkosecki) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala]: ok, patter matched and renamed for clarity

**Pull request merged**